unified-codes/apps/data-service/src/app/schema.ts

```diff
@@ -29,6 +29,10 @@ const typeDefs = gql`
   }
 
   input SearchFilter {
+    "Search by entity code"
+    code: String
```

Comment:
good point :) I don't think that we need a code search - we'd only use a code to return a single entity, I would have thought. Perhaps we should remove as part of #171 and add to the piggybacking?
In some of the designs there are separate search fields for code & description - but since we haven't done that.. shall we remove?

Reply:
Hmm...yeah I kind of think since the codes don't really have a 'structure' as such to them it's not as valuable to search on (compared to a description or properties search)? If certain codes started with say, 'AA' as per some of the designs it would probably be more worthwhile. Onboard with adding removal to the piggybacking 👍